Makefile

```diff
@@ -69,6 +69,16 @@ test:
 push: build
 	@for plugin in $(PLUGIN_YAML_FILES); do \
 		plugin_dir=$$(dirname $${plugin}); \
+		PLUGIN_FULL_NAME=$(shell yq '.name' $${plugin_dir}/buf.plugin.yaml); \
+		PLUGIN_OWNER=`echo "$${PLUGIN_FULL_NAME}" | cut -d '/' -f 2`; \
+		PLUGIN_NAME=`echo "$${PLUGIN_FULL_NAME}" | cut -d '/' -f 3-`; \
+		PLUGIN_VERSION=$(shell yq '.plugin_version' $${plugin_dir}/buf.plugin.yaml); \
+		if [[ -n "$(DOCKER_CACHE_ORG)" ]]; then \
+			CACHE_ARGS=" --cache-from $(DOCKER_CACHE_ORG)/plugins-$${PLUGIN_OWNER}-$${PLUGIN_NAME}:$${PLUGIN_VERSION}"; \
+		fi; \
+		if [[ "$(DOCKER_ORG)" != "bufbuild" ]]; then \
```

Comment:
The only thing that worries me here is if we can't pull the image we'll end up bumping the revision for _all_ pugins. Do you think we should fail hard if for whatever reason docker pull fails? I.e., `not || :;`